config/filesystems.php

```diff
@@ -59,11 +59,31 @@
             'driver' => 's3',
             'key' => env('AWS_ACCESS_KEY_ID'),
             'secret' => env('AWS_SECRET_ACCESS_KEY'),
-            'region' => env('AWS_DEFAULT_REGION'),
-            'bucket' => env('AWS_BUCKET'),
-            'url' => env('AWS_URL'),
+            'region' => env('AWS_DEFAULT_REGION', 'us-east-1'),
```

Comment:
Shouldn't the region also be set from the URL? Or does it not provide that?

Reply:
it does. Check the lastest commit 😉